factory/usr/lib/modules-load.d/ubuntu-core-initramfs.conf

```diff
@@ -2,7 +2,7 @@
 i2c-bcm2708
 nvme
 usbhid
-hid-generic
+xhci-pci
```

Comment:
can you explain this change? does this load all usb drivers?

Reply:
The removed line is because now we have all the hid modules in initramfs because of the change in `bin/ubuntu-core-initramfs` (see call to `dracut-install ... =drivers/hid`).

The new line is to add support for USB 3.x controllers which is what most computers have now.

Reply:
Note that `modules-load.d` is meant to force load modules in the kernel. But we do not need to do that.

Somewhere else, those configuration files are scanned to add the modules into initramfs. But we should only add modules that will be force loaded.

Reply:
It makes sense to of course add support for USB 3.X, very nice change - this doesn't affect support for boards with only USB 2.0 controllers right?

Reply:
That should not.